docs/server/source/_static/arch.svg

```diff
@@ -0,0 +1,2 @@
="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 518.4 653.6 L 485.27 728.14" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 483.56 731.98 L 483.28 725.73 L 485.27 728.14 L 488.4 728 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 264.8 5.6 L 264.8 85.31" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 264.8 89.51 L 262 83.91 L 264.8 85.31 L 267.6 83.91 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 220 5.6 L 220 85.31" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 220 89.51 L 217.2 83.91 L 220 85.31 L 222.8 83.91 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><g transform="translate(147.5,24.5)scale(0.8)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="85" height="12" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; white-space: nowrap;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">BigchainDB API</div></div></foreignObject><text x="43" y="12" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">BigchainDB API</text></switch></g><g transform="translate(271.5,20.5)scale(0.8)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="95" height="26" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; white-space: nowrap;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">MongoDB Cluster<div>Communication</div></div></div></foreignObject><text x="48" y="19" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica">MongoDB Cluster&lt;div&gt;Communication&lt;/div&gt;</text></switch></g><g transform="translate(254.5,719.5)scale(0.8)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="62" height="12" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 12px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; white-space: nowrap; font-weight: bold;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;">IPDB Node</div></div></foreignObject><text x="31" y="12" fill="#000000" text-anchor="middle" font-size="12px" font-family="Helvetica" font-weight="bold">IPDB Node</text></switch></g></g></svg>
```

Comment:
nit pick: may be we don't need to show a distinction b/w service and deployment for simplicity of the logic diagram.

Reply:
We wanted that in this diagram. The earlier one [here](https://medium.com/ipdb-blog/using-the-ipdb-test-network-a615f93d50a9) did not have the separation.